src/Makefile

```diff
@@ -123,24 +124,22 @@ ifeq ($(ARCH),x86-64)
 	sse = yes
 endif
 
-ifeq ($(ARCH),x86-64-modern)
+ifeq ($(ARCH),x86-64-sse3)
 	arch = x86_64
 	prefetch = yes
-	popcnt = yes
 	sse = yes
 	sse3 = yes
-	ssse3 = yes
 endif
 
-ifeq ($(ARCH),x86-64-sse3)
+ifeq ($(ARCH),x86-64-ssse3)
 	arch = x86_64
 	prefetch = yes
 	sse = yes
 	sse3 = yes
 	ssse3 = yes
 endif
 
-ifeq ($(ARCH),x86-64-sse3-popcnt)
+ifeq ($(ARCH),x86-64-ssse3-popcnt)
```

Comment:
AFAIK this is redundant with `sse41-popcnt`. AMD either supports both SSSE3 and SSE4.1 or neither. All Intel supporting POPCNT support SSE4.2. IMO more useful to replace it with `sse3-popcnt`.